boards/BMS/Inc/App/states/App_DriveState.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
+
 #include "App_SharedStateMachine.h"
 
 /**
  * Get a pointer to the Drive State.
  * @return A pointer to the Drive State. THIS SHOULD NOT BE MODIFIED.
  */
-const struct State *App_GetDriveState();
+const struct State *App_GetDriveState(void);
```

Comment:
My understanding is that `()` means accept any parameter so using an explicit `void` is better.

Reply:
yes, absolutely. Good catch.